bigtable/client/rpc_backoff_policy.cc

```diff
@@ -27,14 +27,14 @@ namespace {
 #endif  // BIGTABLE_CLIENT_DEFAULT_MAXIMUM_DELAY
 
 const auto initial_delay = BIGTABLE_CLIENT_DEFAULT_INITIAL_DELAY;
```

Comment:
Can you change initial_delay too for consistency?

Reply:
Done.